scripts/ci-multidisk.sh

```diff
@@ -0,0 +1,42 @@
+#!/bin/bash
```

Comment:
1) Let's put the test in test-cli.py (we need to of course do it in a temporary CODALAB_HOME - see the 'copy' test).

2) Let's avoid having the CONFIG copied here (because then we have to keep it updated).  In the test, the config can be automatically generated by `cl`.